src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -142,9 +148,20 @@
     @Override
     protected int doWork() {
         // Check inputs
-        INPUT.forEach(IOUtil::assertFileIsReadable);
+        IOUtil.assertFilesAreReadable(INPUT);
+        IOUtil.assertFilesAreReadable(INPUT2);
+
         IOUtil.assertFileIsReadable(HAPLOTYPE_MAP);
         if (OUTPUT != null) IOUtil.assertFileIsWritable(OUTPUT);
+
+        if (!INPUT2.isEmpty()) {
+            log.info("INPUT2 is not empty. NOT doing cross-check. Will only compare each SAMPLE in INPUT1 against that sample in INPUT2");
+            if (CROSSCHECK_BY != CrosscheckMetric.DataType.SAMPLE) {
+                log.warn("CROSSCHECK_BY is not SAMPLE, This doesn't make sense in non-crosscheck mode. Ignoring.");
```

Comment:
We do that all the time...I guess I could have another lowercase variable that takes it's value from the right place...but I'm not sure it would make things clearer...

Reply:
changed warning text